### ISSUE TITLE
Multihost with Redis objects and states, log on startup does not match with setup

### ISSUE THREAD

**Issue by gregor-samosir** [buanet/docker-iobroker#122]
Version: v5.0.1beta
Start on an existing installation with redis objects and states.

```------------------------------------------------------------
iobroker    | -----              Debugging information               -----
iobroker    | -----                                                  -----
iobroker    | -----                      System                      -----
iobroker    | -----               arch:      x86_64                  -----
iobroker    | -----                                                  -----
iobroker    | -----                     Versions                     -----
iobroker    | -----               image:                             -----
iobroker    | -----               node:      v12.18.2                -----
iobroker    | -----               npm:       6.14.5                  -----
iobroker    | -----                                                  -----
iobroker    | -----                       ENV                        -----
iobroker    | -----               IOB_ADMINPORT: 8081                    -----
iobroker    | -----               AVAHI:     true                    -----
iobroker    | -----               IOB_MULTIHOST: master                  -----
iobroker    | -----               IOB_OBJECTSDB_HOST: host.docker.internal    -----
iobroker    | -----               IOB_OBJECTSDB_PORT: 6379                    -----
iobroker    | -----               IOB_OBJECTSDB_TYPE: redis                   -----
iobroker    | -----               PACKAGES:  nano                    -----
iobroker    | -----               SETGID:    1000                    -----
iobroker    | -----               SETUID:    1000                    -----
iobroker    | -----               IOB_STATESDB_HOST: host.docker.internal    -----
iobroker    | -----               IOB_STATESDB_PORT: 6379                    -----
iobroker    | -----               IOB_STATESDB_TYPE: redis                   -----
iobroker    | -----               USBDEVICES: none                    -----
iobroker    | -----               ZWAVE:     false                   -----
iobroker    | ------------------------------------------------------------

...............(deleted)

iobroker    | Multihost is set as "master" by ENV and external objects db is set.
iobroker    | Skipping this step...
iobroker    | Done.
iobroker    |  
iobroker    | Multihost is set as "master" by ENV and external states db is set.
iobroker    | Skipping this step...
iobroker    | Done.
iobroker    |  
iobroker    | ENV IOB_OBJECTSDB_TYPE is set and value is different from detected ioBroker installation.
iobroker    | Setting type of objects db to "redis"...
iobroker    | Done.
iobroker    | ENV IOB_OBJECTSDB_HOST is set and value is different from detected ioBroker installation.
iobroker    | Setting host of objects db to "host.docker.internal"...
iobroker    | Done.
iobroker    | ENV IOB_OBJECTSDB_PORT is set and value meets detected ioBroker installation. Nothing to do here.
iobroker    |  
iobroker    | ENV IOB_STATESDB_TYPE is set and value is different from detected ioBroker installation.
iobroker    | Setting type of states db to "redis"...
iobroker    | Done.
iobroker    | ENV IOB_STATESDB_HOST is set and value is different from detected ioBroker installation.
iobroker    | Setting host of states db to "host.docker.internal"...
iobroker    | Done.
iobroker    | ENV IOB_STATESDB_PORT is set and value is different from detected ioBroker installation.
iobroker    | Setting port of states db to "6379"...
iobroker    | Done.
```

Everything is running fine, but the log shows "ENV IOB_OBJECTSDB_TYPE is set and value is **different** from detected ioBroker installation.". Same for _HOST and all _STATESDB_.
It is not different from the existing setup.

On the line "ENV IOB_OBJECTSDB_PORT is set and value meets detected ioBroker installation. Nothing to do here." 
This part looks OK. 

Attached the iobroker.json
[iobroker.txt](https://github.com/buanet/docker-iobroker/files/4865191/iobroker.txt)

**Comment by buanet** [comment]
Confirmed. The solution is still in testing. Will add it to next beta. 
Thank you.

Regards,
André

**Comment by buanet** [comment]
As promised, this is fixed in v5.0.2beta now. Issue stays open until next stable version.  

Regards,
André
